src/components/ProblemAutocompleteModal/ProblemAutocomplete.tsx

```diff
@@ -38,42 +37,36 @@ const ProblemAutocompleteHit = ({
   );
 };
 
-export const ProblemAutocomplete = ({
-  hits,
-  currentRefinement,
-  refine,
-  onProblemSelect,
-  modalIsOpen,
-}) => (
-  <div>
+export const indexName =
+  process.env.NODE_ENV === 'production' ? 'prod_problems' : 'dev_problems';
```

Comment:
should this use `GATSBY_ALGOLIA_INDEX_NAME`?

Reply:
Oh right 😅